frontend/scripts/actions/index.js

```diff
@@ -42,5 +42,6 @@ export default {
   RESET_MAP_DIMENSIONS: 'RESET_MAP_DIMENSIONS',
   SHOW_LINKS_ERROR: 'SHOW_LINKS_ERROR',
   DISPLAY_STORY_MODAL: 'DISPLAY_STORY_MODAL',
-  RESET_TOOL_STATE: 'RESET_TOOL_STATE'
+  RESET_TOOL_STATE: 'RESET_TOOL_STATE',
+  SET_SLIDER_CONTENT: 'home/SET_SLIDER_CONTENT'
```

Comment:
I see the reason for namespacing, but then it makes more sense to have that reflected in the actual action name, like SET_HOMEPAGE_SLIDER_CONTENT: "SET_HOMEPAGE_SLIDER_CONTENT". Otherwise, once you import and use this constant in other places, you actually lose the namespace info, which is helpful.